Console/Command/SpecificProductExport.php

```diff
@@ -0,0 +1,93 @@
+<?php
+declare(strict_types=1);
+
+namespace Aligent\FredhopperIndexer\Console\Command;
+
+use Aligent\FredhopperIndexer\Api\Export\ExporterInterface;
+use Aligent\FredhopperIndexer\Model\Export\Data\ImmediateProducts;
+use Magento\CloudDocker\Cli;
+use Magento\Framework\App\State;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\Area;
+
+class SpecificProductExport extends Command
+{
+    /**
+     * @var State
+     */
+    protected $state;
+
+    /**
+     * @var ExporterInterface
+     */
+    protected $exporter;
+
+    /**
+     * @var ImmediateProducts
+     */
+    protected $immediateProducts;
+
+    /**
+     * @inheritDoc
+     *
+     * @param State $state
+     * @param ExporterInterface $exporter
+     * @param ImmediateProducts $immediateProducts
+     * @param string|null $name
+     */
+    public function __construct(
+        State $state,
+        ExporterInterface $exporter,
+        ImmediateProducts $immediateProducts,
+        string $name = null
```

Comment:
I know it's not technically required for function parameters, but I feel like it's nicer to declare this as `?string`, since `null` is the default